### ISSUE TITLE
false positive: comparison of tuple of integral literals

### ISSUE THREAD

**Issue by pinterior** [microsoft/pyright#8247]
**Describe the bug**

comparing `tuple` of integral literals cause errors.

**Code or Screenshots**

```python
def f():
    return (1,2) < (3,4)
```
```
Operator "<" not supported for types "tuple[Literal[1], Literal[2]]" and "tuple[Literal[3], Literal[4]]"  (reportOperatorIssue)
```
https://pyright-play.net/?locale=en&code=CYUwZgBGAUCUBcAoCKICcQBcCuaB2E0AjADQBMsEAPIQMwkAssiQA

**VS Code extension or command-line**

playground, 1.1.369

**Comment by erictraut** [comment]
Pyright is working as intended here, so this isn't a bug.

Refer to [this documentation](https://microsoft.github.io/pyright/#/type-inference?id=tuple-expressions) for pyright's inference rules for tuple expressions.
